apps/state/lib/state/trip.ex

```diff
@@ -170,6 +171,7 @@ defmodule State.Trip do
       |> build_filters(:direction_id, filters[:direction_id])
       |> build_filters(:route_pattern_id, filters[:route_patterns])
       |> build_filters(:id, filters[:ids])
+      |> build_filters(:revenue, Map.get(filters, :revenue, :REVENUE))
```

Comment:
question: the spec says this is taking a list, so should the default be `[:REVENUE]`?